packages/portal/src/components/item/ItemMediaThumbnails.vue

```diff
@@ -1,32 +1,94 @@
 <template>
   <div
-    class="p-2"
+    class="d-flex flex-column position-relative"
   >
-    <h2>Thumbnails</h2>
     <div
-      class="d-flex overflow-auto"
+      ref="mediaThumbnails"
+      class="media-thumbnails d-flex flex-row flex-lg-column"
     >
```

Comment:
Let's refactor this into an unordered list (`ul`) with list item children (`li`)

Reply:
I've made it an ordered list, since these pages are inherently in sequence. Is there a reason to use ul instead?